src/components/NotificationBox/NotificationBox.tsx

```diff
@@ -0,0 +1,42 @@
+import {useEffect} from 'react';
+
+import {notification} from 'antd';
+
+import {AlertEnum} from '@models/alert';
+
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {clearAlert} from '@redux/reducers/alert';
+
+const NotificationBox: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const alert = useAppSelector(state => state.alert.alert);
+
+  useEffect(() => {
+    if (!alert) {
+      return;
+    }
+
+    let type: string =
```

Comment:
Would be better to have here this somewhere
`
const alertTypes: {[key in AlertEnum]: string} = {
  [AlertEnum.Error]: 'error',
  ...
}`

// usage
const alertType = alertTypes[alert.type] || 'info'